site/src/components/form.md

```diff
@@ -0,0 +1,195 @@
+			name="title"
+			class="input"
+			type="text"
+			placeholder="e.g., Why is the sky blue?"
+		/>
+	</div>
+</div>
+<div class="field">
+	<label class="field-label field-label-lg label" for="sample-form-input-lg"> Input </label>
+	<div class="control">
+		<input
+			id="sample-form-input-lg"
+			name="title"
+			class="input"
+			type="text"
+			placeholder="e.g., Why is the sky blue?"
+		/>
+	</div>
+</div>
+<div class="field display-flex">
+	<div class="control">
+		<button class="button button-primary margin-right-xxs">Submit</button>
+	</div>
+	<div class="control">
+		<button class="button button-danger">Cancel</button>
+	</div>
+</div>
+```
+
+### Modifiers
+
+`.form-addons` is added to attach controls together.
+
+```html
+<div class="field field-addons">
+	<div class="control">
+		<div class="select">
+			<select>
+				<option>$</option>
+				<option>€</option>
+			</select>
+		</div>
+	</div>
+	<div class="control">
+		<input class="input" type="text" placeholder="Transfer money" />
+	</div>
+	<div class="control">
+		<a class="button button-primary"> Transfer </a>
+	</div>
+</div>
+```
+
+`.field-horizontal` is used for displaying form label and field side by side.
+`field-body` is used on the control element container.
+
+```html
+<div class="field field-horizontal">
+	<div class="field-label field-label-sm">
+		<label class="label" for="example-small-label">Search</label>
+	</div>
+	<div class="field-body">
+		<div class="control">
+			<input
+				id="example-small-label"
+				class="input input-sm"
+				type="text"
+				placeholder="www.microsoft.com"
+			/>
+		</div>
+	</div>
+</div>
+<div class="field field-horizontal">
+	<div class="field-label field-label-lg">
+		<label class="label" for="example-large-label">Search</label>
+	</div>
+	<div class="field-body">
+		<div class="control">
+			<input
+				id="example-large-label"
+				class="input input-lg"
+				type="text"
+				placeholder="www.microsoft.com"
+			/>
+		</div>
+	</div>
+</div>
+```
+
+## Form Control
+
+Form control element are wrapped with a `.control` container.
+
+### Modifiers
+
+`.control-icons` is used when the control element contains icons.
+
+```html
+<div class="field">
+	<div class="control control-icons">
+		<input class="input input-sm" placeholder="www.microsoft.com" />
+		<span class="icon border" aria-hidden="true"> </span>
+	</div>
+	<div class="control control-icons margin-top-xxs">
+		<div class="select select-sm">
+			<select>
+				<option>A</option>
+				<option>B</option>
+			</select>
+			<span class="icon border" aria-hidden="true"> </span>
+		</div>
+	</div>
+</div>
+```
+
+`.control-icon-right` is used to display the icon on the right side of the element.
+
+```html
+<div class="field">
+	<div class="control control-icons control-icon-right">
+		<input class="input input-sm" placeholder="www.microsoft.com" />
+		<span class="icon border" aria-hidden="true"> </span>
+	</div>
+</div>
+```
+
+`.is-loading` is used to add a spinner icon.
```

Comment:
can we remove this modifier?

Reply:
Removed. Looks like the autocomplete's loader is separate from this form field loader. If we are planning to add autocomplete (eventually), then perhaps we can add it at that time.